.github/workflows/ci.yml

```diff
@@ -375,4 +375,4 @@ jobs:
           project_name: snuba
           local_image: ghcr.io/getsentry/snuba-ci:${{ github.sha }}
           docker_repo: ghcr.io/getsentry/snuba
-          # TODO: add docker tag to build for
+          docker_password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
should the `action-self-hosted-e2e-tests@5bbd3d6725bbe4bf5cc12f8a3291ce87df6a891f` be the newest commit?